openupgrade_scripts/scripts/website/17.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,370 @@
+---Models in module 'website'---
+new model website.controller.page
+# NOTHING TO DO
+
+---Fields in module 'website'---
+website      / ir.ui.view               / controller_page_ids (one2many): NEW relation: website.controller.page
+website      / website                  / social_tiktok (char)          : NEW hasdefault: default
+website      / website.controller.page  / _inherits                     : NEW _inherits: {'ir.ui.view': 'view_id'}
+website      / website.controller.page  / default_layout (selection)    : NEW selection_keys: ['grid', 'list'], hasdefault: default
+website      / website.controller.page  / is_published (boolean)        : NEW hasdefault: default
+website      / website.controller.page  / menu_ids (one2many)           : NEW relation: website.menu
+website      / website.controller.page  / name_slugified (char)         : NEW isfunction: function, stored
+website      / website.controller.page  / page_name (char)              : NEW required
+website      / website.controller.page  / page_type (selection)         : NEW selection_keys: ['listing', 'single'], hasdefault: default
+website      / website.controller.page  / record_domain (char)          : NEW
+website      / website.controller.page  / view_id (many2one)            : NEW relation: ir.ui.view, required
+website      / website.controller.page  / website_id (many2one)         : NEW relation: website, isrelated: related, stored
+website      / website.menu             / controller_page_id (many2one) : NEW relation: website.controller.page
+website      / website.page             / header_text_color (char)      : NEW
+website      / website.page             / url (char)                    : now required
+# NOTHING TO DO
+
+---XML records in module 'website'---
+NEW ir.actions.act_window: website.action_website_controller_pages_list
+DEL ir.actions.act_window: website.action_show_viewhierarchy
+NEW ir.actions.client: website.action_website_view_hierarchy
+NEW ir.asset: website.s_instagram_page_000_js
+NEW ir.asset: website.s_instagram_page_000_scss
+NEW ir.asset: website.s_text_cover_000_scss
+# NOTHING TO DO
+
+ir.attachment: website.business_conference (noupdate switched)
+ir.attachment: website.header_image_1_default_image (noupdate switched)
```

Comment:
call set_xml_ids_noupdate_value() in pre-migration for all those records with `(noupdate switched)`